app/models/admin_user.rb

```diff
@@ -0,0 +1,2 @@
+class AdminUser < User
+end
```

Comment:
Please use enum for role management rather than STI.

STI is a good thing to know but we usually prefer enum as it is easier to manage in longer run.